src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<blockquote>FAST-OAD automatically add<strong>s</strong> the solver based on what was written in the configuration and automatically sets up and launch the problem.</blockquote><p>--&gt;  FAST-OAD adds the solver based according to the content of the configuration, sets up the problem and runs it.</p><p>("automatically" looks too much to me. That's just a sequence of operations based on user entries)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>